doc/release-notes/11448-api-endpoint-for-analytics-html.md

```diff
@@ -0,0 +1,5 @@
+### Feature Request: API endpoint for analytics.html
+
+New API to get the analytics.html from settings for SPA
```

Comment:
The API returns more than just the analytics code. It returns the header, footer, logo, etc.